etl/steps/data/garden/bp/2023-02-20/energy_mix.meta.yml

```diff
@@ -0,0 +1,61 @@
+dataset:
+  namespace: bp
+  title: Energy mix (BP, 2023)
```

Comment:
I have used `Dataset Name - Source (Year)` formatting in the past, but I see that you have been using `Dataset Name (Source, Year)` in this PR. Is this the preferred format at the moment? I don't have any strong preference, just double checking!

Reply:
Just checked the metadata guidelines on Notion, resolved!

Reply:
Hey @lucasrodes you are also right about being confused here. We have contradictory information in [the Notion metadata guidelines](https://www.notion.so/owid/Metadata-guidelines-29ca6e19b6f1409ea6826a88dbb18bcc?pvs=4) (“Human-readable dataset title (Source name, Year)”.) and on the short description in the grapher web admin ("Short name for this dataset, followed by the source and year. Example: Government Revenue Data – ICTD (2016)").
Given that the dataset title is shown only internally, we can for now allow for both options. But feel free to raise the issue somewhere so we decide about which option is preferrable, thanks.